### ISSUE TITLE
Infinite Scroll doesn't work if the page is not scrollable

### ISSUE THREAD

**Issue by a-magdy** [appbaseio/reactivesearch#369]
Hi guys,

As always thanks for your great work in this library :)

Issue Type: Bug -an edge case with a workaround-

Platform: Web

Description: Infinite scroll doesn't work if the browser page is not scrollable

Screenshots:

- 10 items per page on a big screen:
![skaermbillede 2018-05-17 kl 15 44 37](https://user-images.githubusercontent.com/14139855/40185055-046e12f4-59f2-11e8-9e91-927aab4f6cba.png)

- or only 1 item per page (as an extreme case): 
![skaermbillede 2018-05-17 kl 15 44 15](https://user-images.githubusercontent.com/14139855/40185107-231b8f74-59f2-11e8-8137-c2cc926316a6.png)

Minimal reproduction of the problem with instructions:

- From the booksearch demo, update the `pagination` to false, and update the page size to 10 or less until the scrollbar disappears 
- Try to scroll down to fetch more, but it won't be possible, until you resize the screen, so that the scrollbar appears, so that the infinite loader works

Reactivesearch version: 2.6.2

Browser: Chrome 66

I am not sure whether this is a real issue, but I thought to start a discussion about it, maybe we could come up with a solution for it in the future, like adding a load more button as proposed in this issue: https://github.com/appbaseio/reactivesearch/issues/126 in case the page is not scrollable, and we are not on the last page yet

Thanks :)

**Comment by metagrover** [comment]
Makes sense 👍

`Load More` button should be able to tackle this issue properly.

**Comment by metagrover** [comment]
We used to have a prop called `scrollTarget` which accepted the target container for watching infinite loading on scroll, but we got rid of it in the new architecture. 

It's a trivial implementation if you want to take this up. Open to PRs.

**Comment by dedaimia** [comment]
Thanks, metagrover.  Yes, I'll take a look at it and put in a PR.   May take me a few weeks to get to it.

**Comment by nicholasfay** [comment]
@dedaimia Any update or workaround to this? Currently have a ResultsList inside a component that has its own scroll and would like to use infinite scrolling.

**Comment by metagrover** [comment]
Hi guys, thanks for bringing this up. We will include this in the next release.

```
TODO: Add support for `scrollTarget` prop in result components.
```
cc: @jyash97

**Comment by metagrover** [comment]
Released in `v2.10.0` 🎉
